polus-object-spectral-featurization-plugin/src/mesh.py

```diff
@@ -0,0 +1,165 @@
+import scipy 
+import logging
+import trimesh
+import numpy as np
+from typing import Tuple
+from bfio import BioReader
+from skimage import measure
+
+
+logging.basicConfig(format='%(asctime)s - %(name)s - %(levelname)s - %(message)s',
+                    datefmt='%d-%b-%y %H:%M:%S')
+logger = logging.getLogger('mesh')
+logger.setLevel(logging.INFO)
+
+
+def mesh_spectral_features(vertices, faces, k = 50, scale_invariant = False):
```

Comment:
I think this function is very straight forward and easy to understand, but should also include a docstring. The rule I generally use when creating docstrings is imagine a world where we have full documentation for all of our plugins, and think of the functions in the code someone might see and find useful. If the function is general and useful, I add a docstring even if it's straightforward. I think that is the case with this function.